tower-buffer/src/worker.rs

```diff
@@ -2,7 +2,7 @@ use crate::{
     error::{Closed, Error, ServiceError},
     message::Message,
 };
-use futures_core::{ready, stream::Stream};
```

Comment:
Can we _please_ turn off `#![deny(warnings)]`?

Reply:
Yeah, I think all of them deny only in test cfg.